src/controllers/controllerinputmappingtablemodel.cpp

```diff
@@ -234,6 +242,10 @@ QString ControllerInputMappingTableModel::getDisplayString(const QModelIndex& in
     int column = index.column();
     const MidiInputMapping& mapping = m_midiInputMappings.at(row);
 
+    if (!holds_alternative<ConfigKey>(mapping.control)) {
+        return "";
+    }
```

Comment:
I prefer the std prefix when applicable.
```suggestion
    if (!std::holds_alternative<ConfigKey>(mapping.control)) {
        return QVariant();
    }
```